capa/features/extractors/dnfile/insn.py

```diff
@@ -86,6 +88,40 @@ def extract_insn_api_features(f: CilMethodBody, bb: CilMethodBody, insn: Instruc
             yield API(name_variant), insn.offset
 
 
+def extract_insn_class_features(f: CilMethodBody, bb: CilMethodBody, insn: Instruction) -> Iterator[Tuple[Class, int]]:
+    """parse instruction class features"""
+    if insn.opcode not in (OpCodes.Call, OpCodes.Callvirt, OpCodes.Jmp, OpCodes.Calli):
+        return
+
+    row: Any = resolve_dotnet_token(f.ctx["pe"], Token(insn.operand.value))
+
+    if not isinstance(row, dnfile.mdtable.MemberRefRow):
+        return
+    if not isinstance(row.Class.row, (dnfile.mdtable.TypeRefRow, dnfile.mdtable.TypeDefRow)):
+        return
+
+    yield Class(format_dotnet_classname(row.Class.row.TypeNamespace, row.Class.row.TypeName)), insn.offset
```

Comment:
`Class` could do the formatting (and string handling etc.?)

Reply:
see above.